src/components/Upload.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import styles from './Upload.scss';
+
+const renderDropzone = field => (
```

Comment:
The problem with that is then ES lint starts complaining about props validation and doing props validation for this component somehow feels like an over kill or?